cpp/ql/src/semmle/code/cpp/Macro.qll

```diff
@@ -11,7 +11,7 @@ class Macro extends PreprocessorDirective, @ppd_define {
    */
   override string getHead() { preproctext(underlyingElement(this),result,_) }
 
-  override string getCanonicalQLClass() { result = "Macro" }
+  override string getCanonicalQLClass() { result = "Macro(" + this.getName() + ")" }
```

Comment:
Why do we include additional information just for `Macro`s, `MacroInvocation`s, `Specifier`s and a few other things?